### PR TITLE
add: horizontal mode

### PR DESCRIPTION
**Pull request by wogus3602**
Only Vertical is possible. If you modify it like this, you can also Horizontal. How do you like it?
가능

### CODE REVIEW

File: Sources/VCReorderableStackView.swift [hunk 1]
```diff
@@ -122,29 +122,51 @@ private extension ReorderableStackView {
     }
 
     func reorderViews(with gesture: UILongPressGestureRecognizer) {
-
+        
+        var snapshotViewFrame: CGFloat? = .zero
```
Comment (avreolko) [inline Sources/VCReorderableStackView.swift:126]: We can be more precise with this name.
How about `snapshotViewMidPoint`?

File: Sources/VCReorderableStackView.swift [hunk 2]
```diff
@@ -122,29 +122,51 @@ private extension ReorderableStackView {
...
             : index - 1
-
+        
         guard let reorderView = self.view(for: reorderViewIndex) else { return }
-
-        if midY > max(self.reorderingPoint.y, reorderView.frame.midY)
-        || midY < min(self.reorderingPoint.y, reorderView.frame.midY) {
-
+        
+        var viewFrame: CGFloat = .zero
```
Comment (avreolko) [inline Sources/VCReorderableStackView.swift:150]: Same goes here.

**Review by avreolko — CHANGES_REQUESTED** [review]
Everything is fine, but we need to fix the variable names.

File: Sources/VCReorderableStackView.swift (hunk 1 above)
Comment (wogus3602) [reply Sources/VCReorderableStackView.swift:126]: snapshotViewMidPoint very Nice!

File: Sources/VCReorderableStackView.swift (hunk 2 above)
Comment (wogus3602) [reply Sources/VCReorderableStackView.swift:150]: Nice Naming

**Review by wogus3602 — COMMENTED** [review]
I modified it as you gave me the comments

**Pull request merged**